public/app/features/dashboard/dashgrid/PanelChrome.tsx

```diff
@@ -428,7 +428,6 @@ export class PanelChrome extends Component<Props, State> {
 
     const containerClassNames = classNames({
       'panel-container': true,
-      'panel-container--absolute': true,
```

Comment:
I agree this feels dangerous since it it changes the *view* settings not just the edit settings... but I have not found a dashboard that looked wrong yet.

My guess is that we have so many redundant position elements around panels that the absolute position is not needed anymore

Reply:
i also can't see any issues from my testing, so i'll merge and be on the lookout 👀